### PR TITLE
Feature: color buttons

### PR DESCRIPTION
**Pull request by CalebBassham**
Added support for colored buttons using the color classes.

### CODE REVIEW

File: src/components/button/button.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template lang="html">
-  <div :class="{ 'paper-btn': true, 'btn-block': block, 'btn-small': small, 'btn-large': large, 'disabled': disabled }" :disabled="disabled">
+  <div :class="{ 'paper-btn': true, 'btn-block': block, 'btn-small': small, 'btn-large': large, 'disabled': disabled, 'btn-primary': primary, 'btn-secondary': secondary, 'btn-success': success, 'btn-warning': warning, 'btn-danger': danger }" :disabled="disabled">
```
Comment (TotomInc) [inline src/components/button/button.vue]: Instead of `btn-*`, can't we have a color variable property which takes primary, secondary, ...?

**Review by TotomInc — CHANGES_REQUESTED** [review]

**Comment by TotomInc** [comment]
As said in the review, can't we have a color property which takes as argument `primary`, `secondary`, ...? This avoid creating a lot of variables and keep things simple.

File: src/components/button/button.vue (hunk 1 above)
Comment (CalebBassham) [reply src/components/button/button.vue]: Sure

**Review by TotomInc — APPROVED** [review]
Neat.

**Pull request merged**